python/desc/imsim/camera_readout.py

```diff
@@ -15,17 +15,19 @@
 """
 from __future__ import print_function, absolute_import, division
 import os
-from collections import OrderedDict
+import warnings
+from collections import namedtuple, OrderedDict
 import numpy as np
 import astropy.io.fits as fits
 import astropy.time
 import lsst.afw.geom as afwGeom
 import lsst.afw.image as afwImage
 import lsst.utils as lsstUtils
 from .focalplane_info import FocalPlaneInfo, cte_matrix
+from .imSim import get_logger
```

Comment:
I've noticed some inconsistency in the use of this relative import notation. For example in imSim we have:

from desc.imsim import CosmicRays

Should we care about doing it in a consistent way?

Reply:
I suppose we should care a little.  seems like the changed should be in imSim.py